addons/website_blog/tests/test_website_blog_flow.py

```diff
@@ -58,3 +61,29 @@ def test_website_blog_followers(self):
         self.assertIn(
             self.user_employee.partner_id, test_blog_post.message_partner_ids,
             'website_blog: people commenting a post should follow it afterwards')
+
+    @users('portal_user')
+    def test_blog_comment(self):
+        """Test comment on blog post with attachment."""
+        test_blog = self.env['blog.blog'].sudo().create({
```

Comment:
You could probably extract it in setUpClass, at least in TestWebsiteBlogFlow, and re-use it in test_website_blog_followers to reduce data creation boilerplate.